exe/shopify-toolkit

```diff
@@ -54,6 +54,18 @@ class ShopifyToolkit::CommandLine < Thor
     IRB.conf[:IRB_NAME] = basename
     Result.class_eval { binding.irb(show_code: false) }
   end
+
+  desc "schema_load", 'Load schema from "config/shopify/schema.rb"'
+  def schema_load
+    require "./config/environment"
+    ::Shop.sole.with_shopify_session { ShopifyToolkit::Schema.load! }
```

Comment:
Does this assume that the user has a Shop class? I think this could be added to the assumptions 🤓

Reply:
That's a good call!

Reply:
Fixed 👍